README.md

```diff
@@ -448,6 +438,9 @@ When the code is ready, submit a Pull Request.
 - Prefer code that is readable over one that is "clever".
 - When writing a Git commit message, follow these [guidelines](https://chris.beams.io/posts/git-commit/).
 
+See also Trino's [guidelines](https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md).
```

Comment:
What about creating `DEVELOPMENT.md` or `CONTRIBUTING.md` exclusively for trino-python-client? It can be done as a follow-up and move there all necessary information about testing, contributing?

Reply:
README is still the place where build instructions should live.

Things like code style, release process etc. can go to DEVELOPMENT.md.

I'll send a follow-up PR - thanks for the idea.